content/reference/cd-supporting-new-ci.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Supporting A New CI System
+aliases: ["cd-supporting-new-ci.html"]
+expanded_url: /reference/cd/
+menu:
+  reference:
+    parent: cd
+---
```

Comment:
It might be worth adding `weight: 100` under `parent` (and same indentation as `parent`), so that this article sorts to the bottom of the other CD articles in the table of contents on the left. (100 is just an arbitrary large value). What do you think?

Reply:
Yep. I like that!

Reply:
Hmm adding `weight: 100` is causing this to be the first link rather than last. Actually, any non-zero value for `weight` is doing that. Must be because the other entries don't have a `weight`?

Reply:
Ah, OK. (Sorry, reviewing from my phone and not at the computer to try it myself). Yeah, probably because the other pages don’t have explicit weights. I was hoping we wouldn’t have to add explicit 1, 2, 3, etc. to each of the pages to manually control the order (I did it that way previously). I do wonder if we set all the other pages to have the same explicit weight (e.g. setting them all to 1) if they would still be ordered alphabetically by Hugo, or if that makes Hugo sort them randomly. Anyway, no need to address as part of this PR, I can experiment later.

Reply:
OK sounds good. I'll leave it as-is in this PR.